src/app/pages/PerpetualPage/hooks/usePerpetual_getLatestTradeId.ts

```diff
@@ -0,0 +1,19 @@
+import { useContext, useState } from 'react';
+import { Nullable } from 'types';
+import { RecentTradesContext } from '../components/RecentTradesTable/context';
+import { useDebouncedEffect } from '../../../hooks/useDebouncedEffect';
+
+const BSC_AVERAGE_BLOCK_TIME = 3500; // 3.5s
+
+export const usePerpetual_getLatestTradeId = () => {
+  const [latestTradeId, setLatestTradeId] = useState<Nullable<string>>();
+  const { trades } = useContext(RecentTradesContext);
+
+  useDebouncedEffect(
+    () => setLatestTradeId(trades[0]?.id),
+    [trades],
+    BSC_AVERAGE_BLOCK_TIME,
+  );
```

Comment:
We don't need debounced but throttled.
Debounced delays as long as something comes in.
Throttled instantly calles, and then will only update again after a specific delay.